src/app/Components/MovieList/duck/reducers.js

```diff
@@ -0,0 +1,28 @@
+import { REQUEST_GET_MOVIES, SUCCESS_GET_MOVIES, ERROR_GET_MOVIES } from './actionTypes';
+import { ERROR, SUCCESS, LOADING } from '../../../../common/constants';
+
+const initialState = { status: { state: null, payload: null } };
```

Comment:
It's preferable to leave the word `payload` only for the actions. 
Also, there's no need to wrap it on the `status` object, right?